DataCollector/DoctrineDataCollector.php

```diff
@@ -222,6 +222,9 @@ public function getGroupedQueries()
             });
             $groupedQueries[$connection] = $connectionGroupedQueries;
         }
+        if ((float) $totalExecutionMS === 0.0) {
```

Comment:
Is it safe to compare float values like this? Maybe we can use very small epsilon value e.g. `if ($totalExecutionMS < $eps)` where `$eps = 1e-5` or something like this